vmdb/app/models/mixins/ems_openstack_mixin.rb

```diff
@@ -108,4 +108,30 @@ def verify_credentials(auth_type=nil, options={})
     else;           raise "Invalid OpenStack Authentication Type: #{auth_type.inspect}"
     end
   end
+
+  def stack_create(stack_name, template, options = {})
+    create_options = {:stack_name => stack_name, :template => template.content}.merge(options)
+    openstack_handle.orchestration_service.stacks.new.save(create_options)["id"]
```

Comment:
@chessbyte options are to the `save` method. `new` creates a `Fog::OpenStack::Orchestration::Stack` object which is not a AR model. `save` receives options and passes to provider.